core/client/app/components/gh-ed-editor-scroller.js

```diff
@@ -0,0 +1,67 @@
+import Ember from 'ember';
+import setScrollClassName from 'ghost/utils/set-scroll-classname';
+
+var EditorScroller = Ember.Component.extend({
```

Comment:
I'm looking around, and it doesn't look like this gets used anywhere. Dead code?